infra/gcp/clusters/projects/kubernetes-public/aaa/10-cluster-configuration.tf

```
@@ -154,11 +154,11 @@ resource "google_container_cluster" "cluster" {
     resource_limits {
       resource_type = "cpu"
       minimum = 2
-      maximum = 16
+      maximum = 200
     }
     resource_limits {
       resource_type = "memory"
-      maximum = 64
+      maximum = 500
```

is it possible to have :
```suggestion
      maximum = 256
```